code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -556,6 +556,25 @@
 				else
 					message = "hugs [get_visible_gender() == MALE ? "himself" : get_visible_gender() == FEMALE ? "herself" : "themselves"]."
 
+		if("dab", "dabs")

```

Comment:
Dabbing doesn't exist in 2462. Consider changing it to be the famous Skrellian dance move, the qep. It's quite similar, actually.

Reply:
Fixed